addons/website_slides/controllers/main.py

```diff
@@ -1321,7 +1321,7 @@ def slide_category_search_read(self, fields, domain):
         category_slide_domain = expression.AND([category_slide_domain, [('is_category', '=', True)]])
         can_create = request.env['slide.slide'].check_access_rights('create', raise_exception=False)
         return {
-            'read_results': request.env['slide.slide'].search_read(category_slide_domain, fields),
+            'read_results': request.env['slide.slide'].search_read(category_slide_domain, fields, order='create_date ASC, id'),
```

Comment:
I would assume that by "the last section", François meant "the last section respecting the sequence" and not the actual last chronologically. Otherwise it does not make much sense.

So you should be able to simply remove this change.

EDIT: I see there have been discussions about this during validation, but I'm not sure I understand the point, we should still respect the sequence IMO.